lib/chef/knife/bmcs_server_show.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
+
+require 'chef/knife'
+require 'chef/knife/bmcs_common_options'
+require 'chef/knife/bmcs_helper'
+
+class Chef
+  class Knife
+    # List BMCS instances. Note that this lists all instances in a
+    # compartment, not just those that are set up as Chef nodes.
+    class BmcsServerShow < Knife
+      banner 'knife bmcs server show (options)'
+
+      include BmcsHelper
+      include BmcsCommonOptions
+
+      deps do
+        require 'oraclebmc'
+      end
+
+      option :instance_id,
+             long: '--instance_id LIMIT',
+             description: 'The OCID of the server to display.'
+
+      # Holds information needed to display vnic information
+      class VnicInfo
```

Comment:
Can't you just use the existing Vnic object from the SDK instead of copying into this object?